invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/HomeSearchBar/HomeSearchBar.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react';
+import { goTo } from '../../../../history';
+import { FrontSiteRoutes } from '../../../../routes/urls';
+import { SearchBar } from '../../../../common/components/SearchBar';
+
+export default class HomeSearchBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { query: '' };
+  }
+
+  updateSearchQuery = (value, event) => {
+    this.setState({ query: event.target.value });
+  };
+
+  onSearchExecute = () => {
+    const query = encodeURIComponent(this.state.query);
+    goTo(FrontSiteRoutes.documentsListWithQuery(query));
+  };
+
+  render() {
+    return (
+      <SearchBar
+        currentQueryString={this.state.searchQuery}
+        onInputChange={this.updateSearchQuery}
+        executeSearch={this.onSearchExecute}
+        placeholder={'Search for books, articles, publications...'}
+        className={'fs-headline'}
+        buttonColor={'orange'}
```

Comment:
Maybe we should introduce a `styleConfig.js` where we keep all these hardcoded values i.e. `orange`.

Reply:
orange is semantic-ui variable and it can be changed globally already in the `site.variables` do you think we need a separate config?